### PR TITLE
Improves c-4 code readability

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

I looked up the c4 code the other day to learn about how x-4 was directional now and found out the code is full of variables that were mostly "M" or "AM" or "UM" that made reading it real hard so I made my own vars to make it a little better looking

## Why It's Good For The Game

coding guidelines rule

## Changelog
code improvement

### CODE REVIEW

**Comment by YakumoChen** [comment]
ok I think I'm getting the hang of this hold on

File: code/game/objects/items/grenades/plastic.dm [hunk 1]
```diff
@@ -34,10 +34,10 @@
 	target = null
 	..()
 
-/obj/item/grenade/c4/attackby(obj/item/I, mob/user, params)
-	if(I.tool_behaviour == TOOL_SCREWDRIVER)
+/obj/item/grenade/c4/attackby(obj/item/weapon, mob/user, params)
```
[the remark this answers is not in the document]
Comment (YakumoChen) [reply code/game/objects/items/grenades/plastic.dm:37]: @LemonInTheDark can you switch out with Mothblocks, he made it easier to understand

[the remark this answers is not in the document]
Comment (LemonInTheDark) [reply code/game/objects/items/grenades/plastic.dm:37]: I uh.....
Even in this context the c4 isn't always being hit by a weapon  right? so might as well call it something more general, like item, that we can change the other attackby procs to downline

[the remark this answers is not in the document]
Comment (Mothblocks) [reply code/game/objects/items/grenades/plastic.dm:37]: yeah just make it item im tired

**Pull request merged**